### PR TITLE
Add docker instruction

### PR DESCRIPTION
**Pull request by diamant3**
for issue #24

### CODE REVIEW

File: quickstart.md [hunk 1]
```diff
@@ -58,6 +58,23 @@ On MacOS make sure to install [Brew](https://brew.sh/) first, then run the follo
...
+docker pull pspdev/pspdev:latest
+```
+
+Then you can open the terminal in the directory where you have your code(e.g `/source`) and do something like this:
+
+```shell
+docker run -ti -v .:/source pspdev/pspdev:latest
+```
+
+After that you can cd into the `/source` directory and your pspdev toolchain are ready to use.
```
Comment (sharkwouter) [inline quickstart.md]:
```suggestion
After that you can cd into the `/source` directory and your pspdev toolchain is ready to use.
```

Comment (sharkwouter) [inline quickstart.md]: You might want to move this part down or communicate that the user doesn't have to go through with the next step anymore. Maybe just adding a link to the basic programs or to #Coding at the bottom would do it.

**Comment by sharkwouter** [comment]
Those are some clear instructions, good job. I had a couple of small suggestions, otherwise this looks good. Thanks for making this.

File: quickstart.md (hunk 1 above)
Comment (diamant3) [reply quickstart.md]: I move the docker guide to it's own section wouter, below of the toolchain, what do you think?

Comment (diamant3) [reply quickstart.md]: The toolchain section is the one i edit to make a link to coding section

File: quickstart.md [hunk 2]
```diff
@@ -78,7 +77,28 @@ This can simply be done with `nano ~/.bashrc` (`~/.zprofile` on Mac). To save pr
...
 
-That's it, now the PSPDEV toolchain can be used to build PSP software. Below is the sample code and how to run it. For testing on real hardware, check out [psplink](https://pspdev.github.io/psplinkusb/).
+That's it, now the PSPDEV toolchain can be used to build PSP software. Check out [Coding](#coding) section below for the sample code and how to run it. For testing on real hardware, check out [psplink](https://pspdev.github.io/psplinkusb/).
+
+### Toolchain in Docker
+{: .fs-6 .fw-700 }
+
+PSPDEV has its own [Docker container](https://hub.docker.com/r/pspdev/pspdev) for the toolchain, follow the steps below to set it up.
+
+First, install the docker using the guide from [here](https://docs.docker.com/engine/install/).
```
Comment (sharkwouter) [inline quickstart.md]: This sentence can be in dependencies -> docker

Comment (diamant3) [reply quickstart.md]: Done wouter, is that right? what do you think?

Comment (sharkwouter) [reply quickstart.md]: I added a commit with what I mean. Can you take a look at it and see if that's okay for you?

Comment (diamant3) [reply quickstart.md]: Yes, it's good! wouter, thankss

Comment (diamant3) [reply quickstart.md]: Could i merge this now?

**Comment by sharkwouter** [comment]
Thanks for writing this! I merged it now!

**Pull request merged**